workflow/rocoto/gefs_tasks.py

```diff
@@ -472,7 +486,7 @@ def wavepostpnt(self):
     def extractvars(self):
         deps = []
         if self.app_config.do_wave:
-            dep_dict = {'type': 'task', 'name': 'gefs_wave_post_grid_mem#member#'}
+            dep_dict = {'type': 'metatask', 'name': 'gefs_wave_post_grid_#member#'}
```

Comment:
Needs to match the task name above.
```suggestion
            dep_dict = {'type': 'metatask', 'name': 'gefs_wave_post_grid_mem#member#'}
```